extensions/logging/README.md

```diff
@@ -0,0 +1,158 @@
+# @loopback/extension-logging
+
+This module contains a component provides logging facilities based on
+[Winston](https://github.com/winstonjs/winston) and
+[Fluentd](https://github.com/fluent/fluent-logger-node).
+
+## Stability: :warning:Experimental:warning:
+
+> Experimental packages provide early access to advanced or experimental
+> functionality to get community feedback. Such modules are published to npm
+> using `0.x.y` versions. Their APIs and functionality may be subject to
+> breaking changes in future releases.
+
+## Architecture overview
+
+![logging-component](logging-component.png)
+
+## Installation
+
+```sh
+npm install --save @loopback/extension-logging
+```
+
+## Basic use
+
+The component should be loaded in the constructor of your custom Application
+class.
+
+Start by importing the component class:
+
+```ts
+import {LoggingComponent} from '@loopback/extension-logging';
+```
+
+In the constructor, add the component to your application:
+
+```ts
+this.component(LoggingComponent);
+```
+
+Now your application can add a controller as follows to leverage the logging
+facilities:
+
+```ts
+import {inject} from '@loopback/context';
+import {Logger} from '@loopback/extension-logging';
+
+class MyController {
+  // Inject a winston logger
+  @inject(LoggingBindings.WINSTON_LOGGER)
+  private logger: Logger;
+
+  // http access is logged by a global interceptor
+  @get('/greet/{name}')
+  // log the `greet` method invocations
+  @log()
+  greet(@param.path.string('name') name: string) {
+    return `Hello, ${name}`;
+  }
+
+  @get('/hello/{name}')
+  hello(@param.path.string('name') name: string) {
+    // Use the winston logger explicitly
+    this.logger.log('info', `greeting ${name}`);
+    return `Hello, ${name}`;
+  }
+}
+```
+
+## Configure the logging component
+
+The component contributes bindings with keys declared in `LoggingBindings`
+namespace below:
+
+- FLUENT_SENDER - A fluent sender
+- WINSTON_LOGGER - A winston logger
+- WINSTON_TRANSPORT_FLUENT - A fluent transport for winston
+- WINSTON_INTERCEPTOR - A local interceptor set by `@log` to log method
+  invocations
+- WINSTON_ACCESS_LOGGER - A global interceptor that logs http access with
+  [Morgan](https://github.com/expressjs/morgan) format
+
+The fluent sender and transport for winston can be configured against
+`FLUENT_SENDER`:
```

Comment:
How can the user disable Fluent transport & sender completely, e.g. to use their own transport instead?

Reply:
Fixed the README - it can be disabled by setting `enableFluent` to `false` for the logging component.